lib/board.rb

```diff
@@ -17,4 +17,22 @@ def valid_coordinate?(coordinate)
     end
   end
 
+  def valid_placement?(ship, coordinates)
+    # if time, allow valid coordinates to be passed out of order
+    length_check = ship.length == coordinates.length
+    row_numbers = coordinates.map do |coordinate|
+      coordinate.scan(/\d+/).join.to_f
+    end
+    length = row_numbers.length
```

Comment:
As discussed this works, think we should move forward and refactor later if we have time.